src/broker/broker_admin_pub.c

```diff
@@ -270,6 +272,13 @@ admin_start_cmd (T_BROKER_INFO * br_info, int br_num, int master_shm_id, bool ac
   broker_create_dir (get_cubrid_file (FID_SQL_LOG_DIR, path, BROKER_PATH_MAX));
   broker_create_dir (get_cubrid_file (FID_SLOW_LOG_DIR, path, BROKER_PATH_MAX));
   broker_create_dir (get_cubrid_file (FID_CUBRID_ERR_DIR, path, BROKER_PATH_MAX));
+
+  if (admin_log_file != NULL)
+    {
+      admin_log2 = strdup (admin_log_file);
+      broker_create_dir (dirname (admin_log2));
+    }
```

Comment:
I think that admin_log2 is allocated memory by strdup() so should be freed.

Reply:
It's better to free memory by strdup (), thank you.